### PR TITLE
Export driver

### PR DESCRIPTION
**Pull request by cemezgin**
Exporting driver to enable sqltrace integration

### CODE REVIEW

File: clickhouse.go [hunk 1]
```diff
@@ -6,15 +6,15 @@ import (
...
 
 func init() {
-	sql.Register("chhttp", new(chDriver))
+	sql.Register("chhttp", new(ChDriver))
 }
 
-// chDriver implements sql.Driver interface
-type chDriver struct {
+// ChDriver implements sql.Driver interface
+type ChDriver struct {
```
Comment (DoubleDi) [inline clickhouse.go:13]: Can we name it `Driver` so during usage we specify `clickhouse.Driver` and not `clickhouse.ChDriver`?

File: clickhouse.go [hunk 2]
```diff
@@ -6,15 +6,15 @@ import (
 )
 
 func init() {
-	sql.Register("chhttp", new(chDriver))
+	sql.Register("chhttp", new(ChDriver))
```
Comment (DoubleDi) [inline clickhouse.go:9]: please update tests and examples

Comment (cemezgin) [reply clickhouse.go:9]: you mean as usecase in readme?

**Review by maltsev25 — APPROVED** [review]

**Review by maltsev25 — APPROVED** [review]

**Comment by cemezgin** [comment]
I checked the failing tests but couldn't find any relation between my pr. Do you know what is the issue in there? @DoubleDi

**Review by Trezcool — APPROVED** [review]

**Comment by DoubleDi** [comment]
@cemezgin please update to the latest master branch. The tests are fixed there

File: README.md [hunk 3]
```diff
@@ -267,6 +267,11 @@ func main() {
 	}
 }
 ```
+Use with trace
```
[the remark this answers is not in the document]
Comment (cemezgin) [reply README.md]: okay I put a full datadog tracing example

**Comment by DoubleDi** [comment]
Thanks for your contribution!

**Pull request merged**